nautobot/extras/api/serializers.py

```diff
@@ -1032,11 +1048,33 @@ class Meta:
             "color",
             "description",
             "tagged_items",
+            "content_types",
             "custom_fields",
             "created",
             "last_updated",
         ]
 
+    def validate(self, data):
+        data = super().validate(data)
+
+        errors = {}
+
+        request = self.context["request"]
+
+        # If API Version is >= 1.3
+        if request.major_version > 1 or request.minor_version >= 3:
+            if not data.get("content_types"):
+                errors["errors"] = ["content_types is required"]
```

Comment:
I think we should actually use a different serializer subclass for 1.3 vs 1.2 - take a look at how it's being done in #1420. The 1.2 serializer won't even have a `content_types` field, and the 1.3 serializer will have a `content_types` field with `required=True`.

Reply:
Agreed here. I think for now until we arrive at a better pattern overloading the viewset's `get_serializer_class()` method to inspect `self.request.version` to decide which serializer to return is the best option.